schema.sql

```diff
@@ -39,4 +39,9 @@ species_id INTEGER REFERENCES species(id));
 CREATE TABLE visits(id serial primary key, 
 vets_id integer REFERENCES vets(id), 
 animal_id integer REFERENCES animals(id),
-date_of_visit date);
\ No newline at end of file
+date_of_visit date);
+
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=Run%20the%20following%20query%20to%20add%20an%20extra%20column%20to%20the%20owners%20table%3A), Please add the `email` column to the `owners` table so you can insert email data to it.

Reply:
ok thank you